src/library/scala/collection/immutable/List.scala

```diff
@@ -97,7 +97,7 @@ sealed abstract class List[+A] extends AbstractSeq[A]
 
   // New methods in List
 
-  /** Adds an element at the beginning of this list.
+  /** A copy of this list with an element prepended.
```

Comment:
What is unclear is that two seemingly similar functions have different descriptions that don't really describe their differences (if there are any).  For example, if `+:` and `::` are the same then their descriptions should be the same.  If they are not, then the descriptions should clearly describe how they are different.